.release-notes/4368.md

```diff
@@ -0,0 +1,7 @@
+## Move heap ownership info from chunk to pagemap
+
+An internal runtime change has been made around where/how
+heap chunk ownership information is stored in order to
+improve performance. The tradeoff is that this will now
+use a little more memory in order to realize the performance
+gains.
```

Comment:
```suggestion
An internal runtime change has been made around where/how heap chunk ownership information is stored in order to improve performance. The tradeoff is that this will now use a little more memory in order to realize the performance gains.
```